README.md

```diff
@@ -10,82 +10,180 @@
 
-## Build
-### Download
-To download run the following command:
+<br>
+
+For more info on KVS please see [AWS KVS Documentation](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/what-is-kinesis-video.html) and [Amazon Kinesis Video Streams FAQs](https://aws.amazon.com/kinesis/video-streams/faqs/).
+
+<br>
 
+## Quick Start
+### Required Tools
+The following packages are required to build the SDK libraries. Using a package manager such as _Homebrew_ (Mac), _APT_ (Linux), and _Chocolatey_ (Windows) is the prefered method of installation.
+* C++ Compiler (GNU or Clang recommended)
+* `git`
+* `CMake`
+* `pkg-config` for _Mac_/_Linux_, `pkgconfiglite` for _Windows_
+* `m4`
+* _Windows_ only: `nasm` and `strawberryperl`
+
+<br>
+
+### Install GStreamer
+If building the samples or the kvssink GStreamer plugin, GStreamer libraries are required.
+
+_Mac_
 ```bash
-git clone https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp.git
+brew install gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav
+```
+_Linux_
+```bash
+sudo apt install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-plugins-base-apps gstreamer1.0-plugins-bad gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly gstreamer1.0-tools
+```
+_Windows_
+```bat
+choco install gstreamer --version=1.22.8
+choco install gstreamer-devel --version=1.22.8
+```
+#### Verify GStreamer Installation
+Run the following command to display the GStreamer version to confirm the installation was successful:
+```bash
+gst-launch-1.0 --gst-version
 ```
 
-Note: You will also need to install `pkg-config`, `CMake`, `m4` and a build enviroment. If you are building the GStreamer plugin you will also need GStreamer and GStreamer (Development Libraries).
-
-Refer to the [FAQ](#FAQ) for platform specific instructions.
+<br>
 
-### Configure
+### Download
+```bash
+git clone https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp.git
+```
 
-Prepare a build directory in the newly checked out repository:
+<br>
 
+### Build
+#### Prepare a build directory in the newly checked out repository:
 ```bash
 mkdir -p amazon-kinesis-video-streams-producer-sdk-cpp/build
 cd amazon-kinesis-video-streams-producer-sdk-cpp/build
 ```
+#### Configure the build:
 
-If you are building on Windows you need to generate `NMake Makefiles`, you should run `cmake .. -G "NMake Makefiles"`
+_Mac and Linux_
```

Comment:
Would suggest splitting this into 2 subsections:
1. Build from scratch (build with dependencies=ON)
2. Build using system libraries (build with dependencies=OFF). For this, you would also need to install the necessary dependencies using system package manager

Reply:
Going to add this to the PR description as a future addition to consider, but I'm not set on doing this quite yet since for the smoothest onboarding experience we would recommend to do build dependencies to ensure a supported version of a dependency library is being used.

Reply:
To add to that, it's not as simple as just including the install instructions with the correct versions. Users might already have some of the dependency libraries install with unsupported versions and there would be potential for CMake and/or pkgconfig to link the incorrect ones.